test/prow/env.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// env.go provides a central point to read all environment variables defined by Prow.
+
+package prow
+
+import (
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+type EnvConfig struct {
```

Comment:
Is there a struct defined in Prow that we can use instead of maintaining this struct by ourselves?

Reply:
Good point.
I did some search but k8s test-infra only have the env var names defined as private variables, see https://github.com/kubernetes/test-infra/blob/master/prow/pod-utils/downwardapi/jobspec.go#L80. They also have a function `EnvForType` that can return the environment variables as a map based on the job type, which I think cannot satisfy our needs either.

Reply:
ack. that's sad